src/main/java/file/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package file.writer;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String report, String fileName) {
```

Comment:
Avoid hardcoding the path to resources. It's better to pass the file path as a parameter to the write method and use a relative path when calling this method. Ensure that the input and output files are placed into the 'src/main/resources' folder.